index.php

```diff
@@ -7,8 +7,17 @@
 function getProducts(PDO $db): array
 {
     $query = $db->prepare("SELECT `image`, `product_name`, 
-                                    `business_name`, `currency`, `price`,
+                                    `business_name`, `currency`, `price`
                                 FROM `products`;");
     $query->execute();
     return $query->fetchAll();
-}
\ No newline at end of file
+}
+
+$db = getDb();
+$products = getProducts($db);
+
+echo '<ul>';
```

Comment:
Perhaps try and turn this into a function?

Reply:
Created functions.php file!